### DISCUSSION TITLE
How to remove FPS unlocker?

### DISCUSSION THREAD

**Discussion by AngryBenko** [WakuWakuPadoru/WuWa_Simple_FPSUnlocker#13] [Q&A] [answered]
I'm experiencing some issues while using the FPS unlocker. Do you know how I can revert back to the original settings?

**Comment by WakuWakuPadoru** [comment, marked as answer]
Client > Saved > LocalStorage (Delete this folder)
Client > Saved > Config > WindowsNoEditor > GameUserSettings.ini (Delete this file)

Game will regenerate this files for you after you launch the game.
